empyrical/__init__.py

```diff
@@ -62,3 +63,10 @@
     MONTHLY,
     YEARLY
 )
+
+
+from .perf_attrib import (
+    perf_attrib,
+    compute_exposures,
+    create_perf_attrib_stats,
+)
```

Comment:
Missing a newline here.  You probably can configure your editor to make it always do this automatically for files with .py suffixes.

Reply:
Shall we add flake8 to the build?

Reply:
fixed. it looks like it's already in the build? not sure why it missed the newline:
```
before_script:
  - "flake8 ."
```